src/userbehavior/README.md

```diff
@@ -0,0 +1,58 @@
+# Userbehavior
```

Comment:
Could you replace "the BREACH Framework" with "SOCBED" here? I think we don't have to replace it in the code as well, the readme should suffice.